Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -283,7 +305,40 @@ public Teacher(string firstName, string lastName, string department, DateTime bi
         
         public override string ToString()
         {
-            return "<Teacher>{ firstName: " + firstName + ", lastName: " + lastName + ", department: " + department + ", birthDate: " + birthDate + " }";
+            return $"<Teacher>{{ firstName: {firstName}, lastName: {lastName}, department: {department}, birthDate: {birthDate} }}";
         }
     }
+
+    public class Country
+    {
+        public string Name { get; init; }
+        public City Capital { get; init; }
+        public Politician President { get; init; }
+        public List<State> States { get; init; }
+    }
+
+    public class State
+    {
+        public string Name { get; init; }
+        public City Capital { get; init; }
+        public Politician Governor { get; init; }
+    }
+
+    public class City
+    {
+        public string Name { get; init; }
+        public Politician Mayor { get; init; }
+    }
+
+    public class Politician
```

Comment:
I mean you can keep this type if you want, but don't use real people.

Reply:
Politicians have been replaced with fictional names.